BUILD.gn

```
@@ -307,6 +308,32 @@ source_set("manifests") {
   }
 }
 
+npm_action("electron_version_args") {
+  script = "generate-version-json"
+
+  outputs = [
+    "$target_gen_dir/electron_version.args",
+  ]
+
+  args = [ rebase_path(outputs[0]) ]
```

```suggestion
  args = rebase_path(outputs)
```

`rebase_path` can take an array & return an array